tools/mozdevice/logging.js

```diff
@@ -156,26 +165,26 @@ Logging.prototype.start = function() {
     return;
   }
 
-  var args = [];
   var device = this.device;
   var serial = this.serial;
   var env = process.env;
+  var cmd = 'adb' +
```

Comment:
I don't think we should move away from using the array here as it was more readable.

```js
var args = ['adb'];

if (ADB_HOST) {
  args.push('-H');
  args.push(ADB_HOST);
}

if (ADB_PORT) {
  args.push('-P');
  args.push(ADB_PORT);
}

if (device.properties['ro.build.version.sdk'] >= 21) {
  if (this.clearTimestamp) {
    args.push('-T');
    args.push(this.clearTimestamp);
  } else {
    args.unshift('NOW=$(' + DATE_COMMAND + ');');
    args.push('-T');
    args.push('"$NOW"'); 
  }
}

args.push('logcat');

currentProcess = spawn('bash', ['-c', args.join(' ')], {
 ...
});
```